internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -257,6 +265,32 @@ func (opts *Opts) providerAndRegionToConstant() {
 	opts.Region = strings.ReplaceAll(strings.ToUpper(opts.Region), "-", "_")
 }
 
+func (opts *Opts) defaultValues() error {
+	if !opts.defaultValue {
+		return nil
+	}
+
+	opts.SkipSampleData = true
+	opts.SkipMongosh = true
+	opts.ClusterName = opts.defaultName
+	opts.Provider = aws
+	opts.Region = region
+	opts.tier = atlasM2
+	opts.DBUsername = opts.defaultName
```

Comment:
[nit] it would be ideal to be able to combine flags, ex: `--dbUsername=myUser -Y`, specially since you don't validate they are mutually exclusive

Reply:
it works with the latest commit
```
./bin/mongocli atlas quickstart --default --accessListIp 198.120.1.1 --clusterName TestCluster
You are creating a new Atlas cluster and enabling access to it.

Press [Enter] to use the default values.

Enter [?] on any option to get help.

We are deploying TestCluster...

Creating your cluster... [It's safe to 'Ctrl + C']
\
Now you can connect to your Atlas cluster with: mongosh -u Quickstart-1625581323 -p cl1SSXlMU0xG mongodb+srv://testcluster.ajlj3.mongodb-dev.net
```